prefixcommons/curie_util.py

```diff
@@ -136,7 +143,15 @@ def contract_uri(
 
     """
     if cmaps is None:
-        cmaps = default_curie_maps
+        # TODO warn if not shortest?
```

Comment:
i.e., this implementation is not compatible with the argument `shortest=False` since tries only return the longest